### ISSUE TITLE
Raspberry pi 4 - plex server container wont start - exec user process caused "exec format error"

### ISSUE THREAD

**Issue by jlamalam** [sebgl/htpc-download-box#39]
Hi, 
Thanks for all of this. It's really great and useful. 
I'm having trouble starting the plex container.
This is what I see in the logs:
`standard_init_linux.go:211: exec user process caused "exec format error"`

At first I though it had to due with ARM architecture but I see that is covered on the docker plex image repo

**Comment by jlamalam** [comment]
To anyone seeing themselves getting stuck on this just change the image used to be compatible with ARM and add the respective env variables:

` image: linuxserver/plex:latest`
` - PUID=${PUID} # default user id, defined in .env
   - PGID=${PGID} # default group id, defined in .env
   - VERSION=docker`

**Comment by jbusuttil83** [comment]
yes @jlamalam the plex server in the docker compose is not ARM compatible. I have a fork with a docker compose with a plex server for ARM as I am running the whole setup on a Raspberry Pi
